tests/testthat/test-derive_param_exist_flag.R

```diff
@@ -84,3 +84,26 @@ test_that("derive_param_exist_flag Test 2: error is issued if aval_fun returns w
     fixed = TRUE
   )
 })
+
+
+## derive_param_exist_flag Test 3: error is issued if paramter already exists in dataset ----
+test_that("derive_param_exist_flag Test 3: error is issued if paramter already exists in dataset", {
+  expect_error(
+    actual <- derive_param_exist_flag(
+      dataset = adrs,
+      dataset_adsl = adsl,
+      dataset_add = adrs,
+      filter_add = PARAMCD == "OVR",
+      condition = AVALC == "PD",
+      false_value = "N",
+      set_values_to = vars(
+        PARAMCD = "OVR",
+        ANL01FL = "Y"
+      )
```

Comment:
No need to assign the output to a variable here.

```suggestion
    derive_param_exist_flag(
      dataset = adrs,
      dataset_adsl = adsl,
      dataset_add = adrs,
      filter_add = PARAMCD == "OVR",
      condition = AVALC == "PD",
      false_value = "N",
      set_values_to = vars(
        PARAMCD = "OVR",
        ANL01FL = "Y"
      )
```